### PR TITLE
Correct single to multi-node scaling example

### LINKED ISSUE #11643: Scaling up start-single-node instructions are incorrect
Shaun McVey (smcvey) commented:

Incorrect page: https://www.cockroachlabs.com/docs/stable/cockroach-start-single-node.html#scale-to-multiple-nodes

The instructions in step 5 are incorrect (they do not take a single node's configuration to be the same as if they started a regular cluster).  These should be the correct steps instead:

```
ALTER RANGE default CONFIGURE ZONE USING num_replicas = 3;
ALTER DATABASE system CONFIGURE ZONE USING num_replicas = 5;
ALTER RANGE meta CONFIGURE ZONE USING num_replicas = 5;
ALTER RANGE system CONFIGURE ZONE USING num_replicas = 5;
ALTER RANGE liveness CONFIGURE ZONE USING num_replicas = 5;
ALTER TABLE system.public.replication_constraint_stats CONFIGURE ZONE DISCARD;
ALTER TABLE system.public.replication_constraint_stats CONFIGURE ZONE USING gc.ttlseconds = 600, constraints = '[]', lease_preferences = '[]';
ALTER TABLE system.public.replication_stats CONFIGURE ZONE DISCARD;
ALTER TABLE system.public.replication_stats CONFIGURE ZONE USING gc.ttlseconds = 600, constraints = '[]', lease_preferences = '[]';
```

Jira Issue: DOC-1307

### PR DESCRIPTION
**Pull request by jseldess**
Fixes #11643

### CODE REVIEW

**Comment by jseldess** [comment]
___
*[v20.2/cockroach-start-single-node.md, line 284 at r1](https://reviewable.io/reviews/cockroachdb/docs/11678#-Mjb3lfXBMJy5wTzDtXm:-Mjb3lfYDiuWfXso7eIa:b9ail6q) ([raw file](https://github.com/cockroachdb/docs/blob/447ab67f02a30a92af1509d47460dbb884640165/v20.2/cockroach-start-single-node.md#L284)):*
> ```Markdown
>     ~~~
> 
> 5. Update preconfigured [replication zones](configure-replication-zones.html) to replicate user data 3 times and import internal data 5 times:
> ```

Thanks for reporting this, @smcvey. Where did you get these correct commands? I'd like to loop in an engineer to validate them. 

Also, the intro sentence refers only to updating the replication factor. Can you summarize what the last for commands are about?

**Comment by jseldess** [comment]
___
*[v20.2/cockroach-start-single-node.md, line 284 at r1](https://reviewable.io/reviews/cockroachdb/docs/11678#-Mjb3lfXBMJy5wTzDtXm:-Mjb4EL48ZZfrkLPhPFf:b-o7xvt) ([raw file](https://github.com/cockroachdb/docs/blob/447ab67f02a30a92af1509d47460dbb884640165/v20.2/cockroach-start-single-node.md#L284)):*
<details><summary><i>Previously, jseldess (Jesse Seldess) wrote…</i></summary><blockquote>

Thanks for reporting this, @smcvey. Where did you get these correct commands? I'd like to loop in an engineer to validate them. 

Also, the intro sentence refers only to updating the replication factor. Can you summarize what the last for commands are about? 
</blockquote></details>

Oh, also, I made this change in v20.2, v21.1, and v21.2. Do you know if this is identical across those versions?

**Comment by smcvey** [comment]
@jseldess

> Where did you get these correct commands? I'd like to loop in an engineer to validate them.

I didn't get them from anywhere other that my own testing.  I just compared the output of `show zone configurations;` from a multi-node cluster, and then after turning a single node cluster into a multi-node, compared the outputs of that command.

My assumption is that these commands are meant to make it seem like it was never a single node setup, and should look exactly like a multi-node setup.  The old instructions doesn't quite put the zone configuration settings into the same state - my commands do.

> Also, the intro sentence refers only to updating the replication factor. Can you summarize what the last for commands are about?

The old instructions have this:

```
ALTER TABLE system.public.jobs CONFIGURE ZONE USING num_replicas = 5;
```

The `jobs` table does not have its own zone configuration in a multi-node cluster.  I suspect this was probably correct for a much older version.

As for these:

```
ALTER TABLE system.public.replication_constraint_stats CONFIGURE ZONE USING num_replicas = 5;
ALTER TABLE system.public.replication_stats CONFIGURE ZONE USING num_replicas = 5;
```

They specifically set the num_replicas value at the level of the tables - this is not like the default zone configurations of a multi-node setup where the num_replicas is set at the DATABASE/RANGE system level.  My commands first discard the override then set things correctly.

**Comment by jseldess** [comment]
Thanks, @smcvey. I appreciate the added detail. Do you know if these commands are correct for the versions I've handled in this PR (20.2, 21.1, 21.2)?

**Comment by smcvey** [comment]
I've tested the same instructions for 20.2 and they are correct.  I've no idea about 21.2.  If you can link me a copy of a binary of that version, I can test -  should only take a few minutes.

**Comment by jseldess** [comment]
Thanks, @smcvey. We don't have any official versions of 21.2 to download yet. I think you'd have to build it from source. I'll check if there are, in fact, binaries available somewhere tomorrow.

**Comment by jseldess** [comment]
@mikeCRL, this is an oldish PR I never got over the finish line. Feel free to take it over, hand to someone else, or close.

**Comment by jseldess** [comment]
Hi @lnhsingh! Closing this old PR. If it's still relevant, please feel to have someone reopen and take it over.
